pkg/templates/kubeadm/v1beta3/kubeadm.go

```diff
@@ -48,20 +48,25 @@ const (
 )
 
 const (
-	// fixedEtcdVersion is an etcd version that doesn't have known data integrity and durability bugs
+	// minimumFixedEtcdVersion is an etcd version that doesn't have known data integrity and durability bugs
 	// (see etcdVersionCorruptCheckExtraArgs for more details)
-	fixedEtcdVersion = "3.5.6-0"
```

Comment:
Let's keep this `fixedEtcdVersion` variable and bind it to `3.5.9-0`. Kubernetes very recently switched to 3.5.9 everywhere, so it makes sense that we do the same.

Reply:
I switched back to fixedEtcdVersion and the latest `3.5.10-0`